File: swift/입문/369게임.swift
// 369게임 문제

import Foundation

func solution(_ order:Int) -> Int {
    return String(order).filter { ["3","6","9"].contains(String($0)) }.count
}

// String.contains() 메서드는 Character를 직접 받을 수 있습니다
// in 연산자

File: java/입문/369게임.java
class Solution {
    public int solution(int order) {
        int answer = 0;
        for(char o: String.valueOf(order).toCharArray()) {
            if (o == '3' || o == '6' || o == '9') {
                answer++;
            }
        }
        return answer;
    }
}